.github/workflows/simple-ci.yml

```diff
@@ -0,0 +1,48 @@
+# This is a basic workflow
+
+name: CI-pipeline
+env:
+  APP_NAME    : "converter"
+  PACKAGE_NAME: "converter_app_${{ github.sha }}"
+
+
+# Controls when the workflow will run
+on:
+  push:
+    branches: [ "main", "dev" ]
```

Comment:
As `main` and `dev` are "protected" branches (you don't need to push to them), I think we can remove this line, or exclude them at least.

Reply:
@OleksiiBulba please check 
Added `branches-ignore` [block](https://github.com/MariaMaximova/convert_temp/pull/11/commits/1d0013d6463bc2d0d0720a18d1511fcfb8b5280d#diff-c6209c0c6f59f06d7049138a9573e90c783c168ad994d7a8c9e98e0a8b54b2cf)